### DISCUSSION TITLE
diary/2019/04/05/how-to-study-datascience/

### DISCUSSION THREAD

**Discussion by zzsza** [zzsza/blog-comments#111] [Comments]
#  데이터 사이언티스트가 되기 위해 진행한 다양한 노력들

https://zzsza.github.io/diary/2019/04/05/how-to-study-datascience/

**Comment by zzsza** [comment]
<p>엄청난 나눔이네요. 감사합니다. 잘 보고 갑니다!!</p>

>_This comment was made by Disqus. 2019-06-28 02:14:59 Johnny Koo_

**Comment by zzsza** [comment]
<p>감사합니다! 데이터 관련 직무를 고민하는 문과 졸업생입니다. 이 글을 읽고 용기가 생깁니다. 정말 값진 글입니다.</p>

>_This comment was made by Disqus. 2019-09-01 09:59:46 진준현_

**Comment by zzsza** [comment]
<p>대학원을 가야 하나 막연한 고민이 있었는데, 덕분에 저도 우선 취업을 해서 실무를 해보고 싶다는 방향을 잡아서 데이터 분석 일을 하고 있습니다. 도움이 많이 되었습니다</p>

>_This comment was made by Disqus. 2019-09-05 04:37:46 이마태_

**Comment by zzsza** [comment]
<p>방향을 잡아서 실제 일을 하고 계신다니! 정말 대단하세요 :) 앞으로도 파이팅입니다!</p>

>_This comment was made by Disqus. 2019-09-08 05:31:54 Seong Yun  Byeon_

**Comment by zzsza** [comment]
<p>안녕하세요 :)<br>용기가 생긴다니 다행이네요! 앞으로 더 파이팅이에요 :)</p>

>_This comment was made by Disqus. 2019-09-08 05:32:10 Seong Yun  Byeon_

**Comment by zzsza** [comment]
<p>칭찬해주셔서 감사합니다! :)</p>

>_This comment was made by Disqus. 2019-09-08 05:32:18 Seong Yun  Byeon_

**Comment by zzsza** [comment]
<p>우연히 들어온 블로그에서 너무 큰 Inspiration을 받았습니다 감사합니다.<br>혹시 깃허브 테마는 직접 만드신건지, 테마 같은걸 쓰셨으면 어떤거 쓰셨는지 여쭤봐도 될까요? <br>블로그가 너무 깔끔하고 예뻐요 :)</p>

>_This comment was made by Disqus. 2019-10-08 12:39:44 Huijun Kim_

**Comment by zzsza** [comment]
<p>정말 많은 정보를 얻고 가지만 그만큼 고민도 더 받아갑니다... 전공은 경제지만 원래 컴퓨터 뜯어고치는걸 좋아하기도 했고, 통계와 선형대수를 배우며 부랴부랴 관심이 생겨 마침 대학에 개설된 데이터사이언스전공을 부전공으로 하고 있지만 학부수준에서 배우는 것엔 한계가 너무 명확하더군요.. 대략적인 이미지라도 잡으려 부전공을 신청했는데 데이터는 커녕 초등학생때나 배우던 컴퓨터활용 기초 강의들에 감도 못잡고 있다가 이 블로그에서 정말 많은 정보를 얻고 갑니다. 더 배우려면 대학원에 가야하나 이 실력으로 관련 업종에서 어떻게 일을 하나 고민 중이었는데 사실 글들을 읽다보니 고민이 더욱 깊어집니다...... 벌써 3학년인데 겨우 이 정도 노력해서 될려나 싶기도 하구요..  자괴감과 동기부여를 함께 가져갑니다. 감사합니다</p>

>_This comment was made by Disqus. 2019-10-08 01:57:19 문준혁_

**Comment by zzsza** [comment]
<p>안녕하세요-!<br>제 블로그의 모태는 박민님의 블로그입니다(<a href="http://isme2n.github.io" rel="nofollow noopener" title="isme2n.github.io">isme2n.github.io</a>) 이 블로그 소스를 보고 박민님께 허락을 구를 Github Fork해서 만들었습니다. 검색 기능이 필요하다 생각해서 이 기능은 직접 구현했습니다 :)</p>

>_This comment was made by Disqus. 2019-10-09 11:16:53 Seong Yun  Byeon_

**Comment by zzsza** [comment]
<p>제가 최근 대학생 동아리에서 강의한 방향성에 대한 자료를 보시면 좋을 것 같습니다<br><a href="https://www.slideshare.net/zzsza/ss-173453051" rel="nofollow noopener" title="https://www.slideshare.net/zzsza/ss-173453051">https://www.slideshare.net/...</a></p><p>벌써 3학년이 아니라, 이제 3학년이라 충분히 찾을 수 있다고 생각하시면 좋을 것 같습니다(저는 5학년 때 본격적으로 찾아서 계속 꾸준히 노력했습니다) 생각을 어떻게 하느냐에 따라 자괴감 / 긍정의 마인드로 생각될 수 있습니다. 부정적인 생각을 계속 하다보면 이걸 계속 해야할까?에 대해 계속 생각이 들기도 하구요. 긍정의 마인드로 생각하는 습관도 가지시면 좋을 것 같아요</p>

>_This comment was made by Disqus. 2019-10-09 11:20:02 Seong Yun  Byeon_

**Comment by zzsza** [comment]
<p>안녕하세요. 유튜브를 통해 이 곳 블로그까지 오게되었습니다! 제가 하고자 하는 방향과 너무나 유사해서 저의 롤모델이십니다. 저 또한 글쓴이님처러 데이터를 분석하는 일을 하기위해 열심히 공부중이며 연구보단 현실 문제에 대한 솔루션 도출에 더욱 관심이 많습니다. 현재는 데이터 분석가로 일을하다 머신러닝 엔지니어를 목표로 하고있습니다. 머신러닝 기반의 데이터 분석에 관심이 많습니다 ㅎㅎ 바쁘셔서 답변을 주실지는 모르겠지만 제가 하고자하는 일을 하시기에 궁금한 점을 남깁니다.</p><p>머신러닝을 활용한 데이터 분석가와 머신러닝 엔지니어는 어떤 차이점이 있나요? 분석가는 구현된 머신러닝 알고리즘을 활용하고 분석 결과를 토대로 의사결정을 위한 인사이트 발굴을 하는 것인가요? 머신러닝에 대해 어디까지 활용할 수 있어야 하는지 역량이 궁금합니다.</p><p>저는 머신러닝 중에서도 추천 알고리즘에 관심이 많습니다! 추천 알고리즘을 통한 데이터 분석으로 마케팅 전략 도출을 하는 일을 해보고 싶은데 제가 잡은 키워드가 어떤지 글쓴이님의 의견이 궁금합니다.</p><p>마지막으로 머신러닝 기반 추천 알고리즘을 통한 데이터 분석가로서 필요한 역량이 궁금합니다. 글쓴이님께서 실무를 접하며 느꼈던 점이나 필요한 엔지니어 역량도 말씀해주시면 정말 감사하겠습니다 ㅜㅜ</p><p>긴 글 읽어 주셔서 감사합니다. 앞으로도 블로그에 자주 방문해 열심히 공부하겠습니다.</p>

>_This comment was made by Disqus. 2019-11-01 03:16:29 이현우_

**Comment by zzsza** [comment]
<p>감사합니다!! 너무 큰 도움과 영감을 얻었습니다 ^^</p>

>_This comment was made by Disqus. 2019-12-03 10:13:26 이동희_

**Comment by zzsza** [comment]
<p>너무 잘 읽었습니다 감사합니다~</p>

>_This comment was made by Disqus. 2020-01-10 10:54:29 이동석_

**Comment by zzsza** [comment]
<p>데이터분석을 시작한지 반년이 다되가는데<br>반년전부터 보다가 지금 댓글 남깁니다.<br>어려운 문제가 생기면 나오는 페이지로만 생각하다가<br>많은걸 깨닫고 갑니다!<br>나중에 현업에서 꼭 만나 뵙고 싶습니다.</p>

>_This comment was made by Disqus. 2020-03-13 11:53:58 임영택_

**Comment by zzsza** [comment]
<p>회사에서 딥러닝 개발을 하고 있습니다...사실 개발이라기 보다는 오픈소스 가져다가 입출력을 맞춰주는 건데요.. 저도 딥러닝/데이터 분석 전공자가 아니라서..오픈소스 가져다가 쓰는데..많은 어려움이 있습니다. 왜 저렇게 나올까 하는... 많이 답답하고 있습니다.. 공부해야 하겠다..공부해야 하겠다..이렇게 생각만하고..뭐 부터 해야 할지 까마득했는데.. <br>잘 읽었습니다.. 하나하나씩 공부해야 하겠습니다...</p>

>_This comment was made by Disqus. 2020-06-24 01:47:14 Edward Cho_

**Comment by zzsza** [comment]
<p>구글을 통해 데이터 사이언티스트에 대해 알아보다가 여기까지 오게 되었습니다. 많은 도움을 받고 가는 것 같습니다. 현재 제가 작성자님이 데이터 공부를 시작하셨던 나이인데, 저도 과연 지금해도 되나? 대학원을 준비해야 하나? 많은 고민이 있었는데 많은 힘이 된 것 같습니다. 감사합니다. 저도 작성자님처럼 일단 한 번 부딪혀 보려고 합니다!</p>

>_This comment was made by Disqus. 2020-07-03 09:48:53 Non_Gom_

**Comment by zzsza** [comment]
<p>정말 많은 인사이트를 주는 글 나눔입니다. 정말 감사합니다.</p>

>_This comment was made by Disqus. 2020-08-07 12:33:07 이여름_

**Comment by myojoo** [comment]
멘토롱을 들은 또래 학생이 블로그에 걸어준 링크를 통해 유입되었슴다🍒 
경험 및 커리어패스를 둘러싼 일련의 과정🌶 속의 감정 및 생각을 담담하면서 상세하게 정리해 오픈소스로 남겨주심에 참 감격스럽습니댜. 앞으로 여정 또한 건승하십쇼
